jobs/pulse3d/manifests/overlays/prod/kustomization.yaml

```diff
@@ -7,192 +7,114 @@ resources:
-  patch: |-
-    - op: replace
-      path: /spec/jobTargetRef/template/spec/containers/0/image
-      value: "245339368379.dkr.ecr.us-east-2.amazonaws.com/pulse3d-worker:0.30.4"
-    - op: replace
-      path: /spec/jobTargetRef/template/spec/containers/0/env/1/value
-      value: prod-pulse3d-uploads
-    - op: replace
-      path: /spec/jobTargetRef/template/spec/containers/0/env/2/value
-      value: prod-mantarray-logs
-- target:
-    kind: ScaledJob
-    name: pulse3d-scaledobject-0.30.5
-  patch: |-
-    - op: replace
-      path: /spec/jobTargetRef/template/spec/containers/0/image
-      value: "245339368379.dkr.ecr.us-east-2.amazonaws.com/pulse3d-worker:0.30.5"
-    - op: replace
-      path: /spec/jobTargetRef/template/spec/containers/0/env/1/value
-      value: prod-pulse3d-uploads
-    - op: replace
-      path: /spec/jobTargetRef/template/spec/containers/0/env/2/value
-      value: prod-mantarray-logs
-- target:
-    kind: ScaledJob
-    name: pulse3d-scaledobject-0.31.0
-  patch: |-
-    - op: replace
-      path: /spec/jobTargetRef/template/spec/containers/0/image
-      value: "245339368379.dkr.ecr.us-east-2.amazonaws.com/pulse3d-worker:0.31.0"
-    - op: replace
-      path: /spec/jobTargetRef/template/spec/containers/0/env/1/value
-      value: prod-pulse3d-uploads
-    - op: replace
-      path: /spec/jobTargetRef/template/spec/containers/0/env/2/value
-      value: prod-mantarray-logs
-- target:
-    kind: ScaledJob
-    name: pulse3d-scaledobject-0.32.0
-  patch: |-
-    - op: replace
-      path: /spec/jobTargetRef/template/spec/containers/0/image
-      value: "245339368379.dkr.ecr.us-east-2.amazonaws.com/pulse3d-worker:0.32.0"
-    - op: replace
-      path: /spec/jobTargetRef/template/spec/containers/0/env/1/value
-      value: prod-pulse3d-uploads
-    - op: replace
-      path: /spec/jobTargetRef/template/spec/containers/0/env/2/value
-      value: prod-mantarray-logs
-- target:
-    kind: ScaledJob
-    name: pulse3d-scaledobject-0.32.2
-  patch: |-
-    - op: replace
-      path: /spec/jobTargetRef/template/spec/containers/0/image
-      value: "245339368379.dkr.ecr.us-east-2.amazonaws.com/pulse3d-worker:0.32.2"
-    - op: replace
-      path: /spec/jobTargetRef/template/spec/containers/0/env/1/value
-      value: prod-pulse3d-uploads
-    - op: replace
-      path: /spec/jobTargetRef/template/spec/containers/0/env/2/value
-      value: prod-mantarray-logs
-- target:
-    kind: ScaledJob
-    name: pulse3d-scaledobject-0.32.4
-  patch: |-
-    - op: replace
-      path: /spec/jobTargetRef/template/spec/containers/0/image
-      value: "245339368379.dkr.ecr.us-east-2.amazonaws.com/pulse3d-worker:0.32.4"
-    - op: replace
-      path: /spec/jobTargetRef/template/spec/containers/0/env/1/value
-      value: prod-pulse3d-uploads
-    - op: replace
-      path: /spec/jobTargetRef/template/spec/containers/0/env/2/value
-      value: prod-mantarray-logs
\ No newline at end of file
+  - target:
```

Comment:
Looks like a formatter ran on this, a little hard to tell the diff

Reply:
it was actually an annoying merge diff, vs code updated it's merge editor so I accidentally just kept manually changed the diff.
only change is removal of all old pulse3d versions

Reply:
Can we remove the indents? I think we saw the indents cause issues before for some reason. It was either Kubernetes or argo that didn't like them

Reply:
yep, done